src/libstore/unix/build/local-derivation-goal.cc

```diff
@@ -1847,7 +1850,7 @@ void LocalDerivationGoal::runChild()
                 /* N.B. it is realistic that these paths might not exist. It
                    happens when testing Nix building fixed-output derivations
                    within a pure derivation. */
-                for (auto & path : { "/etc/resolv.conf", "/etc/services", "/etc/hosts" })
+                for (auto & path : { "/etc/resolv.conf", "/etc/services", "/etc/hosts", "/etc/ssl/certs/ca-certificates.crt" })
```

Comment:
This should be unnecessary because we already do 
```
                if (settings.caFile != "" && pathExists(settings.caFile)) {
                    Path caFile = settings.caFile;
                    pathsInChroot.try_emplace("/etc/ssl/certs/ca-certificates.crt", canonPath(caFile, true), true);
                }
```
just below this point.